examples/AlfaKit/Sample/SampleGPS.h

```diff
@@ -18,9 +16,9 @@ void sample_gps_loop() {
 
     if (rn4_modem->gnss.getGNSSData(&data) && data.valid &&
         ((last_send == 0) || (millis() - last_send >= GPS_SEND_INTERVAL))) {
-      last_send = millis();
```

Comment:
Previously, we set 'last_send' to whenever we fetched data, regardless of whether it was complete or not.  Changed to get initial valid data send sooner.